libs/perun/pipes/src/lib/parse-date.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { formatDate } from '@angular/common';
+
+@Pipe({
+  name: 'parseDate'
+})
+export class ParseDatePipe implements PipeTransform {
+
+  transform(value: string): string {
+    if(value.toLowerCase() === 'never'){
+      return value;
+    }
+    return formatDate(value, 'd.MM.yyyy', 'en');
```

Comment:
```suggestion
    return formatDate(value, 'd.M.yyyy', 'en');
```
without this the format of the date was like this 
![image](https://user-images.githubusercontent.com/32871167/99384634-00ffba00-28d0-11eb-93fd-a5aea1d0516c.png)